flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/join/stream/AbstractStreamingJoinOperator.java

```diff
@@ -66,6 +67,10 @@ public abstract class AbstractStreamingJoinOperator extends AbstractStreamOperat
     protected transient JoinConditionWithNullFilters joinCondition;
     protected transient TimestampedCollector<RowData> collector;
 
+    protected transient boolean requiresCopy;
```

Comment:
We don't need this member variable, this only used for minibatch join, it must need to copy.

Reply:
Yes, this could be removed.